lib/checks/navigation/region.js

```diff
@@ -47,6 +52,13 @@ function findRegionlessElms(virtualNode) {
 			dom.getElementByReference(virtualNode.actualNode, 'href')) ||
 		!dom.isVisible(node, true)
 	) {
+		// Mark each parent node as having region descendant
+		let vNode = axe.utils.getNodeFromTree(node);
```

Comment:
From the looks of it, you can use `virtualNode`, which is passed into this function.